pkg/operator/controller/status/controller.go

```diff
@@ -268,6 +288,18 @@ func (r *reconciler) getOperatorState(ingressNamespace, canaryNamespace string)
 		state.IngressControllers = ingressList.Items
 	}
 
+	// Get all CRDs from the gateway.networking.k8s.io group.
```

Comment:
Related to the above:

```suggestion
	// Get all CRDs from the gateway.networking.k8s.io and gateway.networking.x-k8s.io group.
```

Reply:
:heavy_check_mark: